### ISSUE TITLE
Create Athosian Homeworld (Spawn Area)

### ISSUE THREAD

**Issue by Bysokar** [StargateMC/StargateMCPublic#119]
The Athosians need a home... for the same reasons as mentioned for the remaining spawn zones.

**Comment by mainesi** [comment]
Was part done, now wont be finished

Just for the record, i weren't refusing to do Admin work, i was just commenting i dont know perms or understand them for what id think id need. But after that little fit and banning me from the server, i wont be returning.

Told you loads of times i'm bad at coding etc and after all i've done for the server id have expected to be treated better than that

**Comment by Bysokar** [comment]
Three things:

1. I explained how to add perm nodes to admin before you started complaining to players that I'd taken all your rights away and logged off.
2. I've asked you to do a heap of things in the last couple of months that have been flat out ignored... even though they are simple (and well within your abilities).
3. There is no coding involved in NPC creation, nor the server build changes I've asked for, nor is there for permission node changes like I asked for.

I don't know half of this shit either, that's what google is for.

**Comment by mainesi** [comment]
Yeah I know that.

I wernt complaining to players, I was just asking for help cos I didn't know what stuff went with what.

Also mate I never said I wouldn't help with anything. I made one npc a while back and it messed up. I've got my own list here that I wanted to complete before coding work. I don't like having stuff half done. You asked me to do athos and genii, and that's what I was completing.

I couldn't add genii to the universe cos it wernt under protection and I wernt sure how to make a faction without it interfering with my own.

If you don't want me as admin then fine but at least let me continue as a regular player, I still want to enjoy the server
